src/main/java/pixelperfect/event/EventLog.java

```diff
@@ -27,6 +27,7 @@ public EventLog() {
    */
   public void notify(Event event) {
     events.add(event);
+    System.out.println("The ship received a new event: " + event.getDescription());
```

Comment:
Looks good, I'd consider making this NEW EVENT for the next sprint, easier to view.